input/script_utils.go

```diff
@@ -747,7 +756,7 @@ func ReceiverHtlcSpendTimeout(signer Signer, signDesc *SignDescriptor,
 func SecondLevelHtlcScript(revocationKey, delayKey *btcec.PublicKey,
 	csvDelay uint32) ([]byte, error) {
 
-	builder := txscript.NewScriptBuilder()
+	builder := txscript.NewScriptBuilder(txscript.WithScriptAllocSize(80))
```

Comment:
Can we use the constants in `input/size.go` here?

Reply:
Ah, now I understand why I didn't find constants for these values initially: I used `math.MaxUint32` for CSV and CLTV values, which isn't a valid value in the script context and produced a 5 byte varint instead of a 4 byte one.
So now I was able to use the correct constants everywhere.